### PR TITLE
Added deleting by tag functionality.

### PR DESCRIPTION
**Pull request by Cosmic-Goat**
Added a function to delete droplets based on assigned tags using [this API endpoint](https://developers.digitalocean.com/documentation/v2/#deleting-droplets-by-tag). 

I'm pretty new to typescript, so apologies in advance if I missed something. Also my VS code auto formatted [create-api-client.ts](src/utils/create-api-client/create-api-client.ts) to add an extra space, hence the large diff - sorry about that.

Thanks @pjpimentel for the great package!

### CODE REVIEW

**Comment by pjpimentel** [comment]
hey @Cosmic-Goat 

Thank you for that implementation!

I'm going to review and merge it soon as possible, probably the version will be 3.1.1.

**Comment by pjpimentel** [comment]
@Cosmic-Goat
to delivery it faster I will remove the package.json changes, ok?
I'm not sure but I think it can break the install from npm (inside npm only the dist folder is available, so the `build` will fail)

To allow install from from git we can create a issue to work on that.

File: src/modules/droplet/delete-droplet-by-tag/delete-droplet-by-tag.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+export const deleteDropletByTag = ({
+  httpClient,
+}: IContext) => ({
+  tag_name,
+}: IDeleteDropletByTagApiRequest): Promise<Readonly<DeleteDropletByTagResponse>> => {
+    const path = '/droplets';
+    const url = `${path}?tag_name=${tag_name}`;
+
+    return httpClient.delete(url);
```
Comment (pjpimentel) [inline src/modules/droplet/delete-droplet-by-tag/delete-droplet-by-tag.ts]: I think here we could pass the query parameter via the "params", 
before merge I will change it ;)

**Comment by pjpimentel** [comment]
@Cosmic-Goat

I've just deployed it to version 3.1.1 I also changed from deleteDropletByTag to deleteDroplet**s**ByTag.

If you could test and give a feedback I would appreciate!

Ah, about your typescript code: was perfect, thank you!

**Comment by Cosmic-Goat** [comment]
@pjpimentel I tested it and looks good! Also, I didn't mean to add the installation from git change to the master branch / pr - my bad. I meant to add it to a separate branch that so I could have a convenient way to to link to my fork as a dependency until you updated the npm module, which you did really quickly, kudos to you!

**Pull request merged**